src/handlers/payout/post.ts

```diff
@@ -11,7 +21,7 @@ const ItemsToExclude: string[] = ["blockquote"];
  * Incentivize the contributors based on their contribution.
  * The default formula has been defined in https://github.com/ubiquity/ubiquibot/issues/272
  */
-export const incentivizeComments = async () => {
+export async function incentivizeComments() {
```

Comment:
why `export async function incentivizeComments` here? I know it's kind of opinion-based. but want you to follow up on original types.

Reply:
hmmm, maybe it was for using console.trace() to get the track of functions.
@pavlovcik 
RFC

Reply:
We should refactor the log function so that we dont have redundant prefixes we are passing in. Those should be automatically derived from the stack trace. 

https://github.com/ubiquity/ubiquibot/pull/657#discussion_r1302373477